### PR TITLE
18 admin update invoice status

### PR DESCRIPTION
**Pull request by LlamaBack**

### CODE REVIEW

**Comment by LlamaBack** [comment]
merge in other pull request first please!!!

File: app/views/admin/invoices/show.html.erb [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <h1>Invoice #<%= @invoice.id %></h1>
 <div id="invoice_info">
-  <p>Status: <%= @invoice.status %></p>
+    <p>Status: </p><%= form_with model: [:admin, @invoice] do |f| %>
```
Comment (ajkrumholz) [inline app/views/admin/invoices/show.html.erb:3]: 😮 you can pass it an ARRAY of models??? Omg this is incredible news. Today I learned!

**Review by ajkrumholz — APPROVED** [review]
Really good work, super clean and readable! Did NOT know that you could use form_with [@model, @other_model], that really would have gotten me through the struggles I was having with that particular helper. It was working with just the regular advanced routing style, so I moved on, but this is really good to know!

**Review by ajkrumholz — APPROVED** [review]
Really good work, super clean and readable! Did NOT know that you could use form_with [@model, @other_model], that really would have gotten me through the struggles I was having with that particular helper. It was working with just the regular advanced routing style, so I moved on, but this is really good to know!

**Review by Alaina-Noel — APPROVED** [review]

**Pull request merged**